### PR TITLE
fixes #11344 - enable timezone support

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

**Comment by shlomizadok** [comment]
random failure? [test]

File: app/views/foreman_tasks/tasks/_details.html.erb [hunk 1]
```diff
@@ -97,11 +97,11 @@
 </div>
 <div>
   <span class="param-name"><%= _("Started at") %>:</span>
-  <span class="param-value"><%= @task.started_at %></span>
+  <span class="param-value"><%= @task.started_at.in_time_zone %></span>
```
Comment (iNecas) [inline app/views/foreman_tasks/tasks/_details.html.erb]: `try(:in_time_zone)` would be a bit better, as the value does not have to be set always

File: app/views/foreman_tasks/tasks/_details.html.erb [hunk 2]
```diff
@@ -97,11 +97,11 @@
...
 </div>
 <div>
   <span class="param-name"><%= _("Ended at") %>:</span>
-  <span class="param-value"><%= @task.ended_at %></span>
+  <span class="param-value"><%= @task.ended_at.in_time_zone %></span>
```
Comment (iNecas) [inline app/views/foreman_tasks/tasks/_details.html.erb]: ditto

File: app/views/foreman_tasks/tasks/index.html.erb [hunk 3]
```diff
@@ -42,7 +42,7 @@ $(document).on('click', ".table-two-pane td.two-pane-link", function(e) {
       </td>
       <td><%= task.state %></td>
       <td><%= task.result %></td>
-      <td><%= task.started_at %></td>
+      <td><%= task.started_at.in_time_zone %></td>
```
Comment (iNecas) [inline app/views/foreman_tasks/tasks/index.html.erb]: ditto

File: app/views/foreman_tasks/tasks/_details.html.erb (hunk 1 above)
Comment (shlomizadok) [inline app/views/foreman_tasks/tasks/_details.html.erb]: Actually it is always set as in_time_zone falls back to UTC if no time zone is set.

File: app/views/foreman_tasks/tasks/_details.html.erb (hunk 2 above)
Comment (shlomizadok) [inline app/views/foreman_tasks/tasks/_details.html.erb]: ditto ;-)

File: app/views/foreman_tasks/tasks/_details.html.erb (hunk 1 above)
Comment (iNecas) [inline app/views/foreman_tasks/tasks/_details.html.erb]: It's more about the `started_at` being nil, no matter what timezone it's in

Comment (shlomizadok) [inline app/views/foreman_tasks/tasks/_details.html.erb]: Changed.

**Comment by iNecas** [comment]
Thanks @shlomizadok works as expected!

**Pull request merged**